hidtest/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+project(hidtest C)
+
```

Comment:
Not necessary here unless `hidtest` is meant to be a root config. It might actually cause problems.
```suggestion
```

Reply:
> is meant to be a root config

a root config of a _project_
this _is_ a separate project called `hidtest`, that is an example of usage of the `hidapi` _library_ (which is a _separate project_ too)

both happened to be under the same build tree, and that is absolutely fine, CMake is absolutely well designed for it

> It might actually cause problems

I don't believe so, not without misuse of CMake

Reply:
It doesn't add anything, that's the point. Unless you're distributing these sources separately, then it doesn't add anything.